### DISCUSSION TITLE
About projects status

### DISCUSSION THREAD

**Discussion by dodoflix** [jeremylvln/Shulker#460] [General]
My question is actually really simple i just want to know is this project currently maintained? I couldn't find any discord address or etc.. So i had to ask from here. Anyways i love the project and i want to contribute as well, i hope project is currently maintained.

**Comment by jeremylvln** [comment]
Hi @dodoflix !

Thanks a lot for your interest. It means a lot to me!

Do not worry. The project is maintained. I was quite busy recently, but if you look at the commits or the releases, you'll see that it's moving fast :)

I'm still waiting for feedback and testimonies about users using Shulker so I know what to focus on (like improving the documentation as someone suggested not so long ago).

I want to primarily focus on the "core" of Shulker before creating nice add-ons, as I want to use Shilker myself for production use.

Do not hesitate to ask questions or discuss on GitHub Discussions. I love Discord, but for a young project, I prefer having something structured where everyone can find answers to questions.

Many thanks!
